app/scripts/controllers/channelsmodal.js

```diff
@@ -325,10 +357,21 @@ export function ChannelsModalCtrl ($scope, $uibModalInstance, $timeout, Api, Not
     }
   }
 
+  $scope.addHttpMethodsToChannel = function () {
+    $scope.channel.methods = []
+
+    Object.keys($scope.methods).forEach(function (key) {
```

Comment:
No biggie, but it might be a little bit neater to use `filter` instead of `forEach`.